BlocksForAll/SetVariableModViewController.swift

```diff
@@ -28,9 +44,10 @@ class SetVariableModViewController: UIViewController, UITextFieldDelegate{
                 button.isSelected = false
                 button.layer.borderWidth = 0
             } else {
+                deselectAll()
                 button.isSelected = true
```

Comment:
line 49 to 50:
button.layer.borderWidth = 10
button.layer.borderColor = #colorLiteral(red: 0.004859850742, green: 0.09608627111, blue: 0.5749928951, alpha: 1)